namelists/echam/6.3.05p2-awiesm-2.1/cmor/namelist.echam

```diff
@@ -0,0 +1,135 @@
+! This is the CMOR variant of namelist.echam. It contains output
+! which may be used with the DKRZ CMOR-izing tools.
+!
+! P. Gierz
+! Alfred Wegener Institute, Helmholtz Centre for Polar and Marine Research
+! July 2021
```

Comment:
So, we had "slimmed down" namelists for a year, and people are still complaining about too large output 🤦🏻